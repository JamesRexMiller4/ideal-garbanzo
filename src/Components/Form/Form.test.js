import React from 'react';
import {shallow} from 'enzyme';
import Form from './Form';

describe('Form', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Form/>)
  });

  it.skip('should render the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

    describe('Search Bar', () => {
      it.skip('should be able to search for a restaurant', () => {
    
      });
    
      it.skip('should be able to search by name, city, or genre, ')
      });

    describe('Fieldset/Filters', () => {
      it.skip('should be able to filter by state', () => {

      });

      it.skip('should be able to filter by name', () => {

      });

      it.skip('should be able to filter by genre', () => {
        
      });

      it.skip('should be able to filter by city')
    });


});