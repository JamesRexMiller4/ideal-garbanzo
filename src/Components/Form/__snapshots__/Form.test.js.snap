// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form handleClick: should render a different snapshot if advancedSearch is true 1`] = `
<Form
  data={
    Array [
      Object {
        "address1": "201 Waterfront St",
        "attire": "business casual",
        "city": "Oxon Hill",
        "genre": "Steak,American,Contemporary,Seafood,Cafe",
        "hours": "Open Daily 5:30 PM-10:00 PM",
        "id": "f223fdd0-4adc-423e-9747-980a66c256ca",
        "lat": "38.782098",
        "long": "-77.017492",
        "name": "Old Hickory Steakhouse",
        "state": "MD",
        "tags": "Social,Food and Dining,Restaurants,Steakhouses",
        "telephone": "(301) 965-4000",
        "website": "http://www.gaylordnational.com",
        "zip": "20745",
      },
      Object {
        "address1": "303 E Main St",
        "attire": "business casual",
        "city": "Aspen",
        "genre": "Japanese,Sushi,Asian,Contemporary,Seafood",
        "hours": "Open Daily 5:30 PM-9:00 PM",
        "id": "00b35e1a-82b1-4988-b8b9-6df826db2818",
        "lat": "39.190723",
        "long": "-106.82031",
        "name": "Matsuhisa",
        "state": "CO",
        "tags": "Social,Food and Dining,Restaurants,Japanese,Social,Food and Dining,Restaurants,Sushi",
        "telephone": "(970) 544-6628",
        "website": "http://www.matsuhisaaspen.com",
        "zip": "81611",
      },
      Object {
        "address1": "108 3rd St NE",
        "attire": "business casual",
        "city": "Charlottesville",
        "genre": "French,European,Cafe,Continental,American",
        "hours": "Mon-Thu 5:30 PM-9:00 PM; Fri-Sat 5:30 PM-10:00 PM",
        "id": "0f41a3d0-0641-4eef-b7fd-706f083cf0d5",
        "lat": "38.030526",
        "long": "-78.479785",
        "name": "Fleurie Restaurant",
        "state": "VA",
        "tags": "Social,Food and Dining,Restaurants,French",
        "telephone": "(434) 971-7800",
        "website": "http://www.fleurierestaurant.com",
        "zip": "22902",
      },
      Object {
        "address1": "500 Crescent Ct",
        "attire": "business casual",
        "city": "Dallas",
        "genre": "Steak,American",
        "hours": "Mon-Thu 11:00 AM-10:00 PM; Fri 11:00 AM-11:00 PM; Sat 5:00 PM-11:00 PM; Sun 5:00 PM-9:00 PM",
        "id": "0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56",
        "lat": "32.794749",
        "long": "-96.804099",
        "name": "The Capital Grille",
        "state": "TX",
        "tags": "Social,Food and Dining,Restaurants,Steakhouses,Social,Food and Dining,Restaurants,American",
        "telephone": "(214) 303-0500",
        "website": "http://www.thecapitalgrille.com",
        "zip": "75201",
      },
      Object {
        "address1": "8731 International Dr",
        "attire": "Casual",
        "city": "Orlando",
        "genre": "Seafood,International,American,Oysters,Cafe",
        "hours": "Mon-Fri 4:00 PM-10:00 PM; Sat-Sun 2:00 PM-10:00 PM",
        "id": "cd273a24-f8de-44f6-8add-028e22229293",
        "lat": "28.43897",
        "long": "-81.470707",
        "name": "Boston Lobster Feast",
        "state": "FL",
        "tags": "Social,Food and Dining,Restaurants,Seafood,Social,Food and Dining,Restaurants,American",
        "telephone": "(407) 248-8606",
        "website": "http://www.bostonlobsterfeast.com",
        "zip": "32819",
      },
    ]
  }
  resetResults={[MockFunction]}
  setFilteredResults={[MockFunction]}
>
  <form
    className="main-form"
    onSubmit={[Function]}
  >
    <section
      className="regular-search-section"
    >
      <div
        className="select-state-div"
      >
        <label
          htmlFor="select-state"
        >
          Filter by State
        </label>
        <select
          id="select-state"
          name="select"
          onChange={[Function]}
          value=""
        >
          <option
            id=""
            key=""
            value=""
          />
          <option
            id="Alabama"
            key="Alabama"
            value="Alabama"
          >
            Alabama
          </option>
          <option
            id="Alaska"
            key="Alaska"
            value="Alaska"
          >
            Alaska
          </option>
          <option
            id="Arizona"
            key="Arizona"
            value="Arizona"
          >
            Arizona
          </option>
          <option
            id="Arkansas"
            key="Arkansas"
            value="Arkansas"
          >
            Arkansas
          </option>
          <option
            id="California"
            key="California"
            value="California"
          >
            California
          </option>
          <option
            id="Colorado"
            key="Colorado"
            value="Colorado"
          >
            Colorado
          </option>
          <option
            id="Connecticut"
            key="Connecticut"
            value="Connecticut"
          >
            Connecticut
          </option>
          <option
            id="Delaware"
            key="Delaware"
            value="Delaware"
          >
            Delaware
          </option>
          <option
            id="Florida"
            key="Florida"
            value="Florida"
          >
            Florida
          </option>
          <option
            id="Georgia"
            key="Georgia"
            value="Georgia"
          >
            Georgia
          </option>
          <option
            id="Hawaii"
            key="Hawaii"
            value="Hawaii"
          >
            Hawaii
          </option>
          <option
            id="Idaho"
            key="Idaho"
            value="Idaho"
          >
            Idaho
          </option>
          <option
            id="Illinois"
            key="Illinois"
            value="Illinois"
          >
            Illinois
          </option>
          <option
            id="Indiana"
            key="Indiana"
            value="Indiana"
          >
            Indiana
          </option>
          <option
            id="Iowa"
            key="Iowa"
            value="Iowa"
          >
            Iowa
          </option>
          <option
            id="Kansas"
            key="Kansas"
            value="Kansas"
          >
            Kansas
          </option>
          <option
            id="Kentucky"
            key="Kentucky"
            value="Kentucky"
          >
            Kentucky
          </option>
          <option
            id="Louisiana"
            key="Louisiana"
            value="Louisiana"
          >
            Louisiana
          </option>
          <option
            id="Maine"
            key="Maine"
            value="Maine"
          >
            Maine
          </option>
          <option
            id="Maryland"
            key="Maryland"
            value="Maryland"
          >
            Maryland
          </option>
          <option
            id="Massachusetts"
            key="Massachusetts"
            value="Massachusetts"
          >
            Massachusetts
          </option>
          <option
            id="Michigan"
            key="Michigan"
            value="Michigan"
          >
            Michigan
          </option>
          <option
            id="Minnesota"
            key="Minnesota"
            value="Minnesota"
          >
            Minnesota
          </option>
          <option
            id="Mississippi"
            key="Mississippi"
            value="Mississippi"
          >
            Mississippi
          </option>
          <option
            id="Missouri"
            key="Missouri"
            value="Missouri"
          >
            Missouri
          </option>
          <option
            id="Montana"
            key="Montana"
            value="Montana"
          >
            Montana
          </option>
          <option
            id="Nebraska"
            key="Nebraska"
            value="Nebraska"
          >
            Nebraska
          </option>
          <option
            id="Nevada"
            key="Nevada"
            value="Nevada"
          >
            Nevada
          </option>
          <option
            id="New Hampshire"
            key="New Hampshire"
            value="New Hampshire"
          >
            New Hampshire
          </option>
          <option
            id="New Jersey"
            key="New Jersey"
            value="New Jersey"
          >
            New Jersey
          </option>
          <option
            id="New Mexico"
            key="New Mexico"
            value="New Mexico"
          >
            New Mexico
          </option>
          <option
            id="New York"
            key="New York"
            value="New York"
          >
            New York
          </option>
          <option
            id="North Carolina"
            key="North Carolina"
            value="North Carolina"
          >
            North Carolina
          </option>
          <option
            id="North Dakota"
            key="North Dakota"
            value="North Dakota"
          >
            North Dakota
          </option>
          <option
            id="Ohio"
            key="Ohio"
            value="Ohio"
          >
            Ohio
          </option>
          <option
            id="Oklahoma"
            key="Oklahoma"
            value="Oklahoma"
          >
            Oklahoma
          </option>
          <option
            id="Oregon"
            key="Oregon"
            value="Oregon"
          >
            Oregon
          </option>
          <option
            id="Pennsylvania"
            key="Pennsylvania"
            value="Pennsylvania"
          >
            Pennsylvania
          </option>
          <option
            id="Rhode Island"
            key="Rhode Island"
            value="Rhode Island"
          >
            Rhode Island
          </option>
          <option
            id="South Carolina"
            key="South Carolina"
            value="South Carolina"
          >
            South Carolina
          </option>
          <option
            id="South Dakota"
            key="South Dakota"
            value="South Dakota"
          >
            South Dakota
          </option>
          <option
            id="Tennessee"
            key="Tennessee"
            value="Tennessee"
          >
            Tennessee
          </option>
          <option
            id="Texas"
            key="Texas"
            value="Texas"
          >
            Texas
          </option>
          <option
            id="Utah"
            key="Utah"
            value="Utah"
          >
            Utah
          </option>
          <option
            id="Vermont"
            key="Vermont"
            value="Vermont"
          >
            Vermont
          </option>
          <option
            id="Virginia"
            key="Virginia"
            value="Virginia"
          >
            Virginia
          </option>
          <option
            id="Washington"
            key="Washington"
            value="Washington"
          >
            Washington
          </option>
          <option
            id="West Virginia"
            key="West Virginia"
            value="West Virginia"
          >
            West Virginia
          </option>
          <option
            id="Wisconsin"
            key="Wisconsin"
            value="Wisconsin"
          >
            Wisconsin
          </option>
          <option
            id="Wyoming"
            key="Wyoming"
            value="Wyoming"
          >
            Wyoming
          </option>
        </select>
      </div>
      <div
        className="search-bar-div"
      >
        <label
          htmlFor="search-bar"
        >
          Search:
        </label>
        <input
          id="search-bar"
          onChange={[Function]}
          placeholder="Search Bar"
          type="text"
          value=""
        />
        <img
          alt="clear"
          className="clear-icon"
          onClick={[Function]}
          src="close.svg"
        />
        <img
          alt="search"
          className="search-icon"
          onClick={[Function]}
          src="search.svg"
        />
      </div>
      <div
        className="advanced-search-div"
      >
        <h3>
          Advanced Search
        </h3>
        <img
          alt="collapse"
          id="advanced-search-btn"
          onClick={[Function]}
          src="upArrow.svg"
        />
      </div>
    </section>
    <section
      className="advanced-search-section"
    >
      <div
        className="filters-fieldsets-div exposed"
      >
        <div
          className="fieldset-form-div"
          key="Food-fieldset-div"
        >
          <fieldset>
            <legend>
              Food
            </legend>
            <div
              className="checkboxes-div"
            >
              <div
                className="checkbox-div"
                key="Kosher-div"
              >
                <input
                  className="checkbox"
                  id="Kosher"
                  key="Kosher"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Kosher"
                />
                <label
                  htmlFor="Kosher"
                  key="Kosher-label"
                >
                  Kosher
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Oysters-div"
              >
                <input
                  className="checkbox"
                  id="Oysters"
                  key="Oysters"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Oysters"
                />
                <label
                  htmlFor="Oysters"
                  key="Oysters-label"
                >
                  Oysters
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Pasta-div"
              >
                <input
                  className="checkbox"
                  id="Pasta"
                  key="Pasta"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Pasta"
                />
                <label
                  htmlFor="Pasta"
                  key="Pasta-label"
                >
                  Pasta
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Sandwiches-div"
              >
                <input
                  className="checkbox"
                  id="Sandwiches"
                  key="Sandwiches"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Sandwiches"
                />
                <label
                  htmlFor="Sandwiches"
                  key="Sandwiches-label"
                >
                  Sandwiches
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Seafood-div"
              >
                <input
                  className="checkbox"
                  id="Seafood"
                  key="Seafood"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Seafood"
                />
                <label
                  htmlFor="Seafood"
                  key="Seafood-label"
                >
                  Seafood
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Steak-div"
              >
                <input
                  className="checkbox"
                  id="Steak"
                  key="Steak"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Steak"
                />
                <label
                  htmlFor="Steak"
                  key="Steak-label"
                >
                  Steak
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Sushi-div"
              >
                <input
                  className="checkbox"
                  id="Sushi"
                  key="Sushi"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Sushi"
                />
                <label
                  htmlFor="Sushi"
                  key="Sushi-label"
                >
                  Sushi
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Vegetarian-div"
              >
                <input
                  className="checkbox"
                  id="Vegetarian"
                  key="Vegetarian"
                  name="Food"
                  onClick={[Function]}
                  type="checkbox"
                  value="Vegetarian"
                />
                <label
                  htmlFor="Vegetarian"
                  key="Vegetarian-label"
                >
                  Vegetarian
                </label>
              </div>
            </div>
          </fieldset>
        </div>
        <div
          className="fieldset-form-div"
          key="Types-fieldset-div"
        >
          <fieldset>
            <legend>
              Types
            </legend>
            <div
              className="checkboxes-div"
            >
              <div
                className="checkbox-div"
                key="Bakery-div"
              >
                <input
                  className="checkbox"
                  id="Bakery"
                  key="Bakery"
                  name="Types"
                  onClick={[Function]}
                  type="checkbox"
                  value="Bakery"
                />
                <label
                  htmlFor="Bakery"
                  key="Bakery-label"
                >
                  Bakery
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Bistro-div"
              >
                <input
                  className="checkbox"
                  id="Bistro"
                  key="Bistro"
                  name="Types"
                  onClick={[Function]}
                  type="checkbox"
                  value="Bistro"
                />
                <label
                  htmlFor="Bistro"
                  key="Bistro-label"
                >
                  Bistro
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Cafe-div"
              >
                <input
                  className="checkbox"
                  id="Cafe"
                  key="Cafe"
                  name="Types"
                  onClick={[Function]}
                  type="checkbox"
                  value="Cafe"
                />
                <label
                  htmlFor="Cafe"
                  key="Cafe-label"
                >
                  Cafe
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Grill-div"
              >
                <input
                  className="checkbox"
                  id="Grill"
                  key="Grill"
                  name="Types"
                  onClick={[Function]}
                  type="checkbox"
                  value="Grill"
                />
                <label
                  htmlFor="Grill"
                  key="Grill-label"
                >
                  Grill
                </label>
              </div>
            </div>
          </fieldset>
        </div>
        <div
          className="fieldset-form-div"
          key="Styles-fieldset-div"
        >
          <fieldset>
            <legend>
              Styles
            </legend>
            <div
              className="checkboxes-div"
            >
              <div
                className="checkbox-div"
                key="Fusion-div"
              >
                <input
                  className="checkbox"
                  id="Fusion"
                  key="Fusion"
                  name="Styles"
                  onClick={[Function]}
                  type="checkbox"
                  value="Fusion"
                />
                <label
                  htmlFor="Fusion"
                  key="Fusion-label"
                >
                  Fusion
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Traditional-div"
              >
                <input
                  className="checkbox"
                  id="Traditional"
                  key="Traditional"
                  name="Styles"
                  onClick={[Function]}
                  type="checkbox"
                  value="Traditional"
                />
                <label
                  htmlFor="Traditional"
                  key="Traditional-label"
                >
                  Traditional
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Contemporary-div"
              >
                <input
                  className="checkbox"
                  id="Contemporary"
                  key="Contemporary"
                  name="Styles"
                  onClick={[Function]}
                  type="checkbox"
                  value="Contemporary"
                />
                <label
                  htmlFor="Contemporary"
                  key="Contemporary-label"
                >
                  Contemporary
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Eclectic-div"
              >
                <input
                  className="checkbox"
                  id="Eclectic"
                  key="Eclectic"
                  name="Styles"
                  onClick={[Function]}
                  type="checkbox"
                  value="Eclectic"
                />
                <label
                  htmlFor="Eclectic"
                  key="Eclectic-label"
                >
                  Eclectic
                </label>
              </div>
            </div>
          </fieldset>
        </div>
        <div
          className="fieldset-form-div"
          key="Drinks-fieldset-div"
        >
          <fieldset>
            <legend>
              Drinks
            </legend>
            <div
              className="checkboxes-div"
            >
              <div
                className="checkbox-div"
                key="Coffee-div"
              >
                <input
                  className="checkbox"
                  id="Coffee"
                  key="Coffee"
                  name="Drinks"
                  onClick={[Function]}
                  type="checkbox"
                  value="Coffee"
                />
                <label
                  htmlFor="Coffee"
                  key="Coffee-label"
                >
                  Coffee
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Tea-div"
              >
                <input
                  className="checkbox"
                  id="Tea"
                  key="Tea"
                  name="Drinks"
                  onClick={[Function]}
                  type="checkbox"
                  value="Tea"
                />
                <label
                  htmlFor="Tea"
                  key="Tea-label"
                >
                  Tea
                </label>
              </div>
            </div>
          </fieldset>
        </div>
        <div
          className="fieldset-form-div"
          key="Atttire-fieldset-div"
        >
          <fieldset>
            <legend>
              Atttire
            </legend>
            <div
              className="checkboxes-div"
            >
              <div
                className="checkbox-div"
                key="Casual-div"
              >
                <input
                  className="checkbox"
                  id="Casual"
                  key="Casual"
                  name="Atttire"
                  onClick={[Function]}
                  type="checkbox"
                  value="Casual"
                />
                <label
                  htmlFor="Casual"
                  key="Casual-label"
                >
                  Casual
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Business Casual-div"
              >
                <input
                  className="checkbox"
                  id="Business Casual"
                  key="Business Casual"
                  name="Atttire"
                  onClick={[Function]}
                  type="checkbox"
                  value="Business Casual"
                />
                <label
                  htmlFor="Business Casual"
                  key="Business Casual-label"
                >
                  Business Casual
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Smart Casual-div"
              >
                <input
                  className="checkbox"
                  id="Smart Casual"
                  key="Smart Casual"
                  name="Atttire"
                  onClick={[Function]}
                  type="checkbox"
                  value="Smart Casual"
                />
                <label
                  htmlFor="Smart Casual"
                  key="Smart Casual-label"
                >
                  Smart Casual
                </label>
              </div>
              <div
                className="checkbox-div"
                key="Formal-div"
              >
                <input
                  className="checkbox"
                  id="Formal"
                  key="Formal"
                  name="Atttire"
                  onClick={[Function]}
                  type="checkbox"
                  value="Formal"
                />
                <label
                  htmlFor="Formal"
                  key="Formal-label"
                >
                  Formal
                </label>
              </div>
            </div>
          </fieldset>
        </div>
      </div>
    </section>
  </form>
</Form>
`;

exports[`Form should render the snapshot 1`] = `
<Form
  data={
    Array [
      Object {
        "address1": "201 Waterfront St",
        "attire": "business casual",
        "city": "Oxon Hill",
        "genre": "Steak,American,Contemporary,Seafood,Cafe",
        "hours": "Open Daily 5:30 PM-10:00 PM",
        "id": "f223fdd0-4adc-423e-9747-980a66c256ca",
        "lat": "38.782098",
        "long": "-77.017492",
        "name": "Old Hickory Steakhouse",
        "state": "MD",
        "tags": "Social,Food and Dining,Restaurants,Steakhouses",
        "telephone": "(301) 965-4000",
        "website": "http://www.gaylordnational.com",
        "zip": "20745",
      },
      Object {
        "address1": "303 E Main St",
        "attire": "business casual",
        "city": "Aspen",
        "genre": "Japanese,Sushi,Asian,Contemporary,Seafood",
        "hours": "Open Daily 5:30 PM-9:00 PM",
        "id": "00b35e1a-82b1-4988-b8b9-6df826db2818",
        "lat": "39.190723",
        "long": "-106.82031",
        "name": "Matsuhisa",
        "state": "CO",
        "tags": "Social,Food and Dining,Restaurants,Japanese,Social,Food and Dining,Restaurants,Sushi",
        "telephone": "(970) 544-6628",
        "website": "http://www.matsuhisaaspen.com",
        "zip": "81611",
      },
      Object {
        "address1": "108 3rd St NE",
        "attire": "business casual",
        "city": "Charlottesville",
        "genre": "French,European,Cafe,Continental,American",
        "hours": "Mon-Thu 5:30 PM-9:00 PM; Fri-Sat 5:30 PM-10:00 PM",
        "id": "0f41a3d0-0641-4eef-b7fd-706f083cf0d5",
        "lat": "38.030526",
        "long": "-78.479785",
        "name": "Fleurie Restaurant",
        "state": "VA",
        "tags": "Social,Food and Dining,Restaurants,French",
        "telephone": "(434) 971-7800",
        "website": "http://www.fleurierestaurant.com",
        "zip": "22902",
      },
      Object {
        "address1": "500 Crescent Ct",
        "attire": "business casual",
        "city": "Dallas",
        "genre": "Steak,American",
        "hours": "Mon-Thu 11:00 AM-10:00 PM; Fri 11:00 AM-11:00 PM; Sat 5:00 PM-11:00 PM; Sun 5:00 PM-9:00 PM",
        "id": "0b4bfe46-3e60-4de4-82ba-2dd8e5d46b56",
        "lat": "32.794749",
        "long": "-96.804099",
        "name": "The Capital Grille",
        "state": "TX",
        "tags": "Social,Food and Dining,Restaurants,Steakhouses,Social,Food and Dining,Restaurants,American",
        "telephone": "(214) 303-0500",
        "website": "http://www.thecapitalgrille.com",
        "zip": "75201",
      },
      Object {
        "address1": "8731 International Dr",
        "attire": "Casual",
        "city": "Orlando",
        "genre": "Seafood,International,American,Oysters,Cafe",
        "hours": "Mon-Fri 4:00 PM-10:00 PM; Sat-Sun 2:00 PM-10:00 PM",
        "id": "cd273a24-f8de-44f6-8add-028e22229293",
        "lat": "28.43897",
        "long": "-81.470707",
        "name": "Boston Lobster Feast",
        "state": "FL",
        "tags": "Social,Food and Dining,Restaurants,Seafood,Social,Food and Dining,Restaurants,American",
        "telephone": "(407) 248-8606",
        "website": "http://www.bostonlobsterfeast.com",
        "zip": "32819",
      },
    ]
  }
  resetResults={[MockFunction]}
  setFilteredResults={[MockFunction]}
>
  <form
    className="main-form"
    onSubmit={[Function]}
  >
    <section
      className="regular-search-section"
    >
      <div
        className="select-state-div"
      >
        <label
          htmlFor="select-state"
        >
          Filter by State
        </label>
        <select
          id="select-state"
          name="select"
          onChange={[Function]}
          value=""
        >
          <option
            id=""
            key=""
            value=""
          />
          <option
            id="Alabama"
            key="Alabama"
            value="Alabama"
          >
            Alabama
          </option>
          <option
            id="Alaska"
            key="Alaska"
            value="Alaska"
          >
            Alaska
          </option>
          <option
            id="Arizona"
            key="Arizona"
            value="Arizona"
          >
            Arizona
          </option>
          <option
            id="Arkansas"
            key="Arkansas"
            value="Arkansas"
          >
            Arkansas
          </option>
          <option
            id="California"
            key="California"
            value="California"
          >
            California
          </option>
          <option
            id="Colorado"
            key="Colorado"
            value="Colorado"
          >
            Colorado
          </option>
          <option
            id="Connecticut"
            key="Connecticut"
            value="Connecticut"
          >
            Connecticut
          </option>
          <option
            id="Delaware"
            key="Delaware"
            value="Delaware"
          >
            Delaware
          </option>
          <option
            id="Florida"
            key="Florida"
            value="Florida"
          >
            Florida
          </option>
          <option
            id="Georgia"
            key="Georgia"
            value="Georgia"
          >
            Georgia
          </option>
          <option
            id="Hawaii"
            key="Hawaii"
            value="Hawaii"
          >
            Hawaii
          </option>
          <option
            id="Idaho"
            key="Idaho"
            value="Idaho"
          >
            Idaho
          </option>
          <option
            id="Illinois"
            key="Illinois"
            value="Illinois"
          >
            Illinois
          </option>
          <option
            id="Indiana"
            key="Indiana"
            value="Indiana"
          >
            Indiana
          </option>
          <option
            id="Iowa"
            key="Iowa"
            value="Iowa"
          >
            Iowa
          </option>
          <option
            id="Kansas"
            key="Kansas"
            value="Kansas"
          >
            Kansas
          </option>
          <option
            id="Kentucky"
            key="Kentucky"
            value="Kentucky"
          >
            Kentucky
          </option>
          <option
            id="Louisiana"
            key="Louisiana"
            value="Louisiana"
          >
            Louisiana
          </option>
          <option
            id="Maine"
            key="Maine"
            value="Maine"
          >
            Maine
          </option>
          <option
            id="Maryland"
            key="Maryland"
            value="Maryland"
          >
            Maryland
          </option>
          <option
            id="Massachusetts"
            key="Massachusetts"
            value="Massachusetts"
          >
            Massachusetts
          </option>
          <option
            id="Michigan"
            key="Michigan"
            value="Michigan"
          >
            Michigan
          </option>
          <option
            id="Minnesota"
            key="Minnesota"
            value="Minnesota"
          >
            Minnesota
          </option>
          <option
            id="Mississippi"
            key="Mississippi"
            value="Mississippi"
          >
            Mississippi
          </option>
          <option
            id="Missouri"
            key="Missouri"
            value="Missouri"
          >
            Missouri
          </option>
          <option
            id="Montana"
            key="Montana"
            value="Montana"
          >
            Montana
          </option>
          <option
            id="Nebraska"
            key="Nebraska"
            value="Nebraska"
          >
            Nebraska
          </option>
          <option
            id="Nevada"
            key="Nevada"
            value="Nevada"
          >
            Nevada
          </option>
          <option
            id="New Hampshire"
            key="New Hampshire"
            value="New Hampshire"
          >
            New Hampshire
          </option>
          <option
            id="New Jersey"
            key="New Jersey"
            value="New Jersey"
          >
            New Jersey
          </option>
          <option
            id="New Mexico"
            key="New Mexico"
            value="New Mexico"
          >
            New Mexico
          </option>
          <option
            id="New York"
            key="New York"
            value="New York"
          >
            New York
          </option>
          <option
            id="North Carolina"
            key="North Carolina"
            value="North Carolina"
          >
            North Carolina
          </option>
          <option
            id="North Dakota"
            key="North Dakota"
            value="North Dakota"
          >
            North Dakota
          </option>
          <option
            id="Ohio"
            key="Ohio"
            value="Ohio"
          >
            Ohio
          </option>
          <option
            id="Oklahoma"
            key="Oklahoma"
            value="Oklahoma"
          >
            Oklahoma
          </option>
          <option
            id="Oregon"
            key="Oregon"
            value="Oregon"
          >
            Oregon
          </option>
          <option
            id="Pennsylvania"
            key="Pennsylvania"
            value="Pennsylvania"
          >
            Pennsylvania
          </option>
          <option
            id="Rhode Island"
            key="Rhode Island"
            value="Rhode Island"
          >
            Rhode Island
          </option>
          <option
            id="South Carolina"
            key="South Carolina"
            value="South Carolina"
          >
            South Carolina
          </option>
          <option
            id="South Dakota"
            key="South Dakota"
            value="South Dakota"
          >
            South Dakota
          </option>
          <option
            id="Tennessee"
            key="Tennessee"
            value="Tennessee"
          >
            Tennessee
          </option>
          <option
            id="Texas"
            key="Texas"
            value="Texas"
          >
            Texas
          </option>
          <option
            id="Utah"
            key="Utah"
            value="Utah"
          >
            Utah
          </option>
          <option
            id="Vermont"
            key="Vermont"
            value="Vermont"
          >
            Vermont
          </option>
          <option
            id="Virginia"
            key="Virginia"
            value="Virginia"
          >
            Virginia
          </option>
          <option
            id="Washington"
            key="Washington"
            value="Washington"
          >
            Washington
          </option>
          <option
            id="West Virginia"
            key="West Virginia"
            value="West Virginia"
          >
            West Virginia
          </option>
          <option
            id="Wisconsin"
            key="Wisconsin"
            value="Wisconsin"
          >
            Wisconsin
          </option>
          <option
            id="Wyoming"
            key="Wyoming"
            value="Wyoming"
          >
            Wyoming
          </option>
        </select>
      </div>
      <div
        className="search-bar-div"
      >
        <label
          htmlFor="search-bar"
        >
          Search:
        </label>
        <input
          id="search-bar"
          onChange={[Function]}
          placeholder="Search Bar"
          type="text"
          value=""
        />
        <img
          alt="clear"
          className="clear-icon"
          onClick={[Function]}
          src="close.svg"
        />
        <img
          alt="search"
          className="search-icon"
          onClick={[Function]}
          src="search.svg"
        />
      </div>
      <div
        className="advanced-search-div"
      >
        <h3>
          Advanced Search
        </h3>
        <img
          alt="expand"
          id="advanced-search-btn"
          onClick={[Function]}
          src="downArrow.svg"
        />
      </div>
    </section>
    <section
      className="advanced-search-section"
    >
      <div
        className="filters-fieldsets-div"
      />
    </section>
  </form>
</Form>
`;
